CONTRIBUTING.md

```diff
@@ -83,3 +83,14 @@ Contributing to ecosystem projects is just as important as contributing to Solid
 If you haven't found any interesting information on this page then we encourage you to start hacking at a Solid related utility or package that does. Building useful tools for fellow OSS ecosystem and Solid users enhances the whole platform.
 
 We can't wait to see what you build!
+
+## Building Solid
+
+This repository uses [pnpm](https://pnpm.io/) and
+[Turbo](https://turbo.hotwired.dev/).
```

Comment:
Wrong turbo.. should be this one: https://turborepo.org/

Reply:
Whoops, indeed! I wasn't sure whether links were worth adding, but this kind of confirms that they are. 😅 Fixed!